sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 4.0.0-beta.6 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
 ## 4.0.0-beta.5 (Unreleased)
```

Comment:
we did not release a beta for last release.
and we have already merged [this](https://github.com/Azure/azure-sdk-for-java/pull/27666) so we can close this?

Reply:
Oh it looks like this was created for a branch other than `main`. I recall the EngSys folks were going to work on a way to re-run some release steps, so it might be a byproduct of that.